### DISCUSSION TITLE
Why am I getting an APIERROR when publishing to Netlify?

### DISCUSSION THREAD

**Discussion by DamonCharlesRoberts** [quarto-dev/quarto-cli#6893] [Q&A] [answered]
### Description

I have a RevealJS presentation that I am trying to publish with Netlify. However, once Quarto has successfully built the HTML file and is saying that it will publish it to Netlify, I get the following message:

![image](https://github.com/quarto-dev/quarto-cli/assets/55409288/358c2335-ec7d-45ad-adae-63671eb8aa7e)

I am using Quarto 1.3.450. Any insight into what is causing this would be very helpful!

**Comment by cscheid** [comment, marked as answer]
This could be a `quarto publish` bug. Could you create a GitHub issue documenting every command you've used? Thanks.

**Comment by DamonCharlesRoberts** [comment]
Created an Issue here: https://github.com/quarto-dev/quarto-cli/issues/6899
